src/model.js

```diff
@@ -1034,8 +1034,8 @@ class Model {
       }
     });
 
-    if (!_.includes(['and', 'overwrite'], this.options?.whereMergeStrategy)) {
-      throw new Error(`Invalid value ${this.options?.whereMergeStrategy} for whereMergeStrategy. Allowed values are 'and' and 'overwrite'.`);
+    if (!_.includes(['and', 'overwrite'], this.options && this.options.whereMergeStrategy)) {
```

Comment:
This has tests somewhere right? Because this doesn't seem right to me, && just return true and false right? So it would never include 'and' or 'overwrite'

Reply:
&& returns the first operand if it is false, and the second otherwise value, `1 && 2 === 2`
A feature very specific to JavaScript

Reply:
Aah, today I learned